src/engine/engine.rs

```diff
@@ -44,7 +63,7 @@ pub trait Pool: HasName + HasUuid {
     /// Returns a list of device nodes corresponding to devices actually added.
     /// Returns an error if a blockdev can not be added because it is owned
     /// or there was an error while reading or writing a blockdev.
-    fn add_blockdevs(&mut self, paths: &[&Path], force: bool) -> EngineResult<Vec<PathBuf>>;
+    fn add_blockdevs(&mut self, paths: &[&Path], force: bool) -> EngineResult<Vec<DevUuid>>;
```

Comment:
This seems like a good change...header comment needs updating.

Reply:
ok